<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerConnectAndDisplayModuleWidget</class>
 <widget class="QWidget" name="qSlicerConnectAndDisplayModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="ConnectorPropertyFrame">
     <property name="title">
      <string/>
     </property>
     <widget class="SlicerVideoWidget" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>1280</width>
        <height>720</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>40</y>
        <width>205</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="ConnectorPortLabel">
           <property name="text">
            <string>  Port:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ConnectorPortEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="FrequencyLabel">
           <property name="text">
            <string>Frequency:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="FrameFrequency">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Active</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="ConnectorStateCheckBox">
             <property name="minimumSize">
              <size>
               <width>19</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Start Video</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="StartVideoCheckBox">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="ConnectorNameLabel">
           <property name="text">
            <string>VideoInputs</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="qMRMLNodeComboBox" name="NodeSelector" native="true">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>10</height>
            </size>
           </property>
           <property name="nodeTypes" stdset="0">
            <stringlist>
             <string>vtkMRMLVideoNode</string>
            </stringlist>
           </property>
           <property name="addEnabled" stdset="0">
            <bool>false</bool>
           </property>
           <property name="removeEnabled" stdset="0">
            <bool>false</bool>
           </property>
           <property name="showChildNodeTypes" stdset="0">
            <bool>false</bool>
           </property>
           <property name="selectNodeUponCreation" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
  <class>SlicerVideoWidget</class>
  <extends>QWidget</extends>
  <header>Widgets/SlicerVideoWidget.h</header>
  <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ConnectorStateCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
